<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: rgb(0, 90, 137);
	color: white
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>رویداد مالی خود را ثبت کنید</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CostLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #f9f9f9;
    color: #2c3e50;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #3498db;
    background-color: #ecf6fc;
}

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>مبلغ (ریال):</string>
   </property>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #f9f9f9;
    color: #2c3e50;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px;
    font-size: 14px;
	qproperty-font: &quot;FontAwesome&quot;;
}

QComboBox:focus {
    border: 1px solid #3498db;
    background-color: #ecf6fc;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
    border-left: 1px solid #bdc3c7;
    border-radius: 0px 6px 6px 0px;
    background-color: #e0e0e0;
}

QComboBox::down-arrow {
    image: url(icons/down-arrow.png);  /* مسیر نسبی نسبت به main.py */
    width: 12px;
    height: 12px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>حساب بانکی:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>تاریخ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>توضیحات (اختیاری):</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>201</width>
     <height>79</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: #f9f9f9;
    color: #2c3e50;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px;
    font-size: 14px;
}

QTextEdit:focus {
    border: 1px solid #3498db;
    background-color: #ecf6fc;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmEventButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 16pt &quot;.AppleSystemUIFont&quot;;
    background-color:rgb(109, 171, 231);
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: rgb(109, 160, 200);
}

</string>
   </property>
   <property name="text">
    <string>ثبت</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backbutton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 11pt &quot;.AppleSystemUIFont&quot;;
    background-color:rgb(109, 171, 231);
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: rgb(109, 160, 200);
}

</string>
   </property>
   <property name="text">
    <string>بازگشت</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #f9f9f9;
    color: #2c3e50;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 1px solid #3498db;
    background-color: #ecf6fc;
}

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>نوع رویداد:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #f9f9f9;
    color: #2c3e50;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px;
    font-size: 14px;
	qproperty-font: &quot;FontAwesome&quot;;
}

QComboBox:focus {
    border: 1px solid #3498db;
    background-color: #ecf6fc;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
    border-left: 1px solid #bdc3c7;
    border-radius: 0px 6px 6px 0px;
    background-color: #e0e0e0;
}

QComboBox::down-arrow {
    image: url(icons/down-arrow.png);  /* مسیر نسبی نسبت به main.py */
    width: 12px;
    height: 12px;
}





</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;.AppleSystemUIFont&quot;;</string>
   </property>
   <property name="text">
    <string>دسته بندی:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="categoryComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #f9f9f9;
    color: #2c3e50;
    border: 1px solid #bdc3c7;
    border-radius: 6px;
    padding: 6px;
    font-size: 14px;
	qproperty-font: &quot;FontAwesome&quot;;
}

QComboBox:focus {
    border: 1px solid #3498db;
    background-color: #ecf6fc;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px;
    border-left: 1px solid #bdc3c7;
    border-radius: 0px 6px 6px 0px;
    background-color: #e0e0e0;
}

QComboBox::down-arrow {
    image: url(icons/down-arrow.png);  /* مسیر نسبی نسبت به main.py */
    width: 12px;
    height: 12px;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
